### PR TITLE
Support Points() in subtasks and partial-points OK verdicts

### LINKED ISSUE #57: Allow entering subtask scores
This feature is useful so that if TLX support "tcframe generator" later (upload testcases using only tcframe runner.cpp), TLX can automatically set the subtask configuration, including the scores.

### LINKED ISSUE #119: Support IOI-style partial score grading
Right now, the built-in grader can only grade ICPC-style verdicts (AC/WA for non-crashing output). For custom scorers, only AC/WA is valid as well.

We want the grader to support partial scores with OK [score] verdicts.

### PR DESCRIPTION
**Pull request by fushar**
Resolve #57 
Resolve #119

### CODE REVIEW

**Comment by fushar** [comment]
@jonathanirvings you will love this

File: docs/api-ref/api-ref.rst [hunk 1]
```diff
@@ -289,22 +289,39 @@ Defines the constraints to be imposed to the :ref:`input/output variables <api-r
...
+        CONS(A <= B && B <= 1000);
+        CONS(graphDoesNotHaveCycles());
+    }
+
+.. sourcecode:: cpp
+
+    void Subtask1() {
+        Points(70);
+
+        CONS(A <= B && B <= 1000);
```
Comment (jonathanirvings) [inline docs/api-ref/api-ref.rst]: Ini emang sengaja diulang gini?

File: docs/api-ref/api-ref.rst [hunk 2]
```diff
@@ -289,22 +289,39 @@ Defines the constraints to be imposed to the :ref:`input/output variables <api-r
...
+
+    void Constraints() {
+        CONS(A <= B && B <= 1000);
+        CONS(graphDoesNotHaveCycles());
+    }
+
+.. sourcecode:: cpp
+
+    void Subtask1() {
+        Points(70);
```
Comment (jonathanirvings) [inline docs/api-ref/api-ref.rst]: Ini emang sengaja maksimum nya enggak 100 gini?

Comment (fushar) [inline docs/api-ref/api-ref.rst]: Yap

File: docs/api-ref/api-ref.rst (hunk 1 above)
Comment (fushar) [inline docs/api-ref/api-ref.rst]: Ya, mereka bertiga beda sections

**Pull request merged**